module/setup.py

```diff
@@ -3,9 +3,10 @@
 from distutils.command.install_data import install_data
 import pprint
 import glob
+import os
```

Comment:
Even though `import os` works, it is probably better to `import os.path` explicitly ("explicit is better than implicit" from python zen list: https://www.python.org/dev/peps/pep-0020/ )